src/server/dfly_main.cc

```diff
@@ -701,6 +702,43 @@ void PrintBasicUsageInfo() {
   std::cout << endl;
 }
 
+extern char** environ;
+
+void ParseFlagsFromEnv() {
+  const auto& flags = absl::GetAllFlags();
+  for (char** env = environ; *env != nullptr; env++) {
+    constexpr string_view kPrefix = "DFLY_";
+    string_view environ_var = *env;
+    if (absl::StartsWith(environ_var, kPrefix)) {
+      // Per 'man environ', environment variables are included with their values
+      // in the format "name=value". Need to strip them apart, in order to work with flags object
+      pair<string_view, string_view> environ_pair =
+          absl::StrSplit(absl::StripPrefix(environ_var, kPrefix), '=');
```

Comment:
Doesn't it happen with the pair receiver already?

Reply:
> // Stores first two split strings as the members in a std::pair. Any split
// strings beyond the first two are omitted because std::pair can hold only two
// elements.

True... my bad 😓

Reply:
confirming its working here:

<img width="1135" alt="Screenshot 2023-09-12 at 12 02 46" src="https://github.com/dragonflydb/dragonfly/assets/5136677/989aeee2-0c76-4ee4-a7d0-bdf3a9774c70">